.solr_wrapper

```diff
@@ -0,0 +1,9 @@
+# Place any default configuration for solr_wrapper here
+# port: 8983
+
+instance_dir: tmp/solr-development
+
+download_dir: tmp/solr_wrapper
+
+version: 6.5.1
```

Comment:
I'll push up this change once I poke around in travis to see if the cache details are working or not for the current push/pr.